_config.yml

```diff
@@ -35,6 +35,7 @@ collections:
 # Scripts
 scripts:
   - /assets/js/build/main.js
+  - /assets/js/build/touchpoints_translations.js
```

Comment:
I think the commit prior to this one (402d028d024406a67d54302288b5a8ee023ed29f) would have been fine to get Circle passing again? Problem with this is that this configuration controls which scripts are output by default in the loop in `_includes/scripts.html`, so we'd be including the script twice on help pages, and also including it on all other pages where we're not expecting it to be added.